src/tenant.ts

```diff
@@ -65,15 +65,16 @@ export class Tenant {
         );
     }
 
-    modifyTenant(passwd: string, extra?: object) {
+    modifyTenant(passwd: string, fabricName='_system', extra={}) {
```

Comment:
This API is not related to tenant anymore. Better is to write this in the `User` class of jsc8 and add test case for the same too.

Reply:
@lohaniabhishek ok but this is related to tenant password change then also it should moved to user class?Can you again look at the ticket  once https://github.com/Macrometacorp/C8Platform/issues/1121

Reply:
@lohaniabhishek in user already this method is available but martin wants this method to be changes with this end point. What do you suggest?

Reply:
@lohaniabhishek @VikasPatel made the required changes according to the updated ticket.